composer.json

```
@@ -24,7 +24,8 @@
     "require": {
         "php": "^7.4|^8.0",
         "ext-json": "*",
-        "php-amqplib/php-amqplib": "^2.11|^3.0",
+        "ext-sockets": "*",
+        "php-amqplib/php-amqplib": "^3.1.2",
```

Is there a reason to make `3.1.2` as a minimal version, or `3.1.0` will be ok too?
```suggestion
        "php-amqplib/php-amqplib": "^3.1.0",
```

---

this version alsoworks, but why use it and not the last one avaliable?